zhaquirks/sonoff/snzb02d.py

```diff
@@ -105,5 +115,27 @@ class AttributeDefs(BaseAttributeDefs):
         translation_key="display_unit",
         fallback_name="Display unit",
     )
+    .number(
+        CustomSonoffCluster.AttributeDefs.temperature_calibration.name,
+        CustomSonoffCluster.cluster_id,
+        min_value=-50,
+        max_value=50,
+        step=0.1,
+        unit=TemperatureUnit,
+        multiplier=0.01,
+        translation_key="temperature_calibration",
+        fallback_name="Temperature calibration",
+    )
+    .number(
+        CustomSonoffCluster.AttributeDefs.humidity_calibration.name,
+        CustomSonoffCluster.cluster_id,
+        min_value=-50,
+        max_value=50,
+        step=0.1,
+        unit=PERCENTAGE,

```

Comment:
Hmm, technically, the unit for the calibration won't be a percentage, but percentage points.
Don't really HA uses `pp` as a unit for entities like this though. I do see at least one HA entity using "%", so this is likely *fine*.